microservice/subscription/src/main/java/com/cumulocity/microservice/subscription/annotation/EnableMicroserviceSubscriptionConfiguration.java

```diff
@@ -87,6 +90,9 @@ public MicroserviceMetadataRepresentation metadata(Environment environment) thro
                 return MicroserviceMetadataRepresentation.microserviceMetadataRepresentation()
                         .requiredRoles(MoreObjects.firstNonNull(manifest.getRequiredRoles(), ImmutableList.<String>of()))
                         .roles(MoreObjects.firstNonNull(manifest.getRoles(), ImmutableList.<String>of()))
+                        .extensibleDeviceRegistration(
```

Comment:
Maybe you should extend ManifestRepresentation ?

Reply:
BTW Microservice manifest already have a concept of com.cumulocity.model.application.microservice.Extension
Maybe you can extend the representation to support it ?

Reply:
thanks, I will check this

Reply:
using List<ExtensionRepresentation> extensions in MicroserviceMetadataRepresentation and extended the MicroserviceManifestRepresentation from core with this PR: https://github.softwareag.com/IOTA/cumulocity-core/pull/339